Monika After Story/game/script-topics.rpy

```diff
@@ -14195,3 +14195,28 @@ label monika_translating_poetry:
         m 5ekbsa "It would be nice if you could read some of them for me sometime..."
         m 5rkbsu "You'd have to translate them for me first, though~"
     return
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_diving",
+            category=["nature"],
+            prompt="Diving",
+            random=True
+        )
+     )
+label monika_diving:
+    m 3eub "[player], have you heard about diving?"
+    m 1eub "this is scuba diving with special equipment."
+    m 3wub "It's very exciting. Just imagine you are diving."
+    m 3eub "There can see a wonderful underwater world full of different marine life."
+    m 7eub "You can see: schools of fish, coral reefs, and sea greens."
+    m 4eub "In addition to predatory fish, sharks can be seen there."
```

Comment:
I'm pretty sure sharks are predatory animals.

Reply:
There are some species of sharks that feed on plankton.
Such as a whale shark, although it reaches 15-20 meters, it is not dangerous. It does not attack anyone at all, feeds on plankton and swims majestically in surface waters well warmed by the sun.

Reply:
Generally speaking, when people think of sharks, I'm pretty sure they imagine the predatory kind. True, there are some sharks that are non predatory and harmless, but there are also plenty that are dangerous.